metadata-ingestion/src/datahub/ingestion/source/glue.py

```diff
@@ -560,15 +560,25 @@ def get_column_type(
         "varchar": StringTypeClass,
     }
 
+    field_starts_type_mapping = {
+        "array": ArrayTypeClass,
+        "set": ArrayTypeClass,
+        "map": MapTypeClass,
+        "struct": MapTypeClass,
+        "varchar": StringTypeClass,
+        "decimal": NumberTypeClass,
+    }
+
+    type_class = None
     if field_type in field_type_mapping.keys():
```

Comment:
Note: if you want to check if a key is in a dict, there's no need for `.keys()` – see https://stackoverflow.com/questions/1602934/check-if-a-given-key-already-exists-in-a-dictionary